forge/ee/lib/billing/index.js

```diff
@@ -47,17 +45,23 @@ module.exports.init = function (app) {
             app.log.info(`Adding Project ${project.id} to Subscription for team ${team.hashid}`)
 
             if (projectItem) {
-                const metadata = projectItem.metadata ? projectItem.metadata : {}
-                console.log('updating metadata', metadata)
+                const metadata = subscription.metadata ? subscription.metadata : {}
+                // console.log('updating metadata', metadata)
                 metadata[project.id] = 'true'
-                console.log(metadata)
+                // console.log(metadata)
                 const update = {
                     quantity: projectItem.quantity + 1,
-                    proration_behavior: 'always_invoice',
-                    metadata: metadata
+                    proration_behavior: 'always_invoice'
                 }
                 // TODO update meta data?
-                stripe.subscriptionItems.update(projectItem.id, update)
+                try {
+                    await stripe.subscriptionItems.update(projectItem.id, update)
+                    await stripe.subscriptions.update(subscription.id, {
+                        metadata: metadata
+                    })
+                } catch (error) {
+                    app.log.info(`Problem adding project to subscription\n${error.message}`)
```

Comment:
This looks like we swallow any failures to update the subscription and let things continue? Should this fail back and be reported to the user?

Also, not sure `info` is the right level of logging failures like this - unless it is completely benign.

Reply:
Moving over to `warn` level. I can propagate the error up, would just need to work out what the UI should show. 
At the moment a failure in the first instance means we don't bill but the project exists consuming resources.
For the second the project is not removed from a bill but is shutdown.

I think we should probably change the order on the remove path, so a project is only removed from billing if it is shutdown.

Reply:
Errors now propagate up.